src/main/wlp/server.xml

```diff
@@ -1,8 +1,7 @@
 <server description="new server">
 
     <!-- Enable features -->
-<featureManager>
-       <feature>jms-2.0</feature>

```

Comment:
Do the wasJmsClient-2.0 and wasJmsServer-1.0 take care of the jms-2.0 feature? I think that's why I took it out in the first place. https://www.ibm.com/support/knowledgecenter/SSAW57_liberty/com.ibm.websphere.wlp.nd.multiplatform.doc/ae/twlp_msg.html